src/connectors/infinite-hits/connectInfiniteHits.ts

```diff
@@ -63,7 +63,7 @@ const connectInfiniteHits: InfiniteHitsConnector = (
       escapeHTML = true,
       transformItems = (items: any[]) => items,
       showPrevious: hasShowPrevious = false,
-    } = widgetParams || {};
+    } = widgetParams || ({} as Partial<InfiniteHitsRendererWidgetParams>);
```

Comment:
which one requires this to be partial? Is it something we throw an error for?

Reply:
It is equivalent as `InfiniteHitsConnectorParams` https://github.com/algolia/instantsearch.js/blob/c7c53de037959070422c7be85882423078dc1253/src/connectors/infinite-hits/connectInfiniteHits.ts#L23-L25
I will change to `InfiniteHitsConnectorParams` to be more coherent with the other widgets.